libs/cart/src/cart-state.module.ts

```diff
@@ -15,7 +15,7 @@ import { DaffCartPaymentMethodsEffects } from './effects/cart-payment-methods.ef
 
 @NgModule({
   imports: [
-    StoreModule.forFeature('cart', daffCartReducers),
+    StoreModule.forFeature('cart', daffCartReducers()),
```

Comment:
Does this need a call? Can the underlying function just be a reference?

Reply:
changed